File: android/vpn/app/src/main/java/com/clostra/newnode/vpn/statistics/DataVolume.java
package com.clostra.newnode.vpn.statistics;

public class DataVolume {
    private final long myDirect;
    private final long myPeer;

    public DataVolume(long direct, long peer) {
        myDirect = direct;
        myPeer = peer;
    }

    public long getDirect() {
        return myDirect;
    }

    public long getPeer() {
        return myPeer;
    }
}

File: ios/NewNode VPN/NewNode VPN/DataVolume.swift
//
//  DataVolume.swift
//  NewNode VPN
//
//  Created by Mikhail Koroteev on 06/02/21.
//  Copyright © 2021 Clostra. All rights reserved.
//

import Foundation

class DataVolume {
    var direct: Int64 = 0
    var peer: Int64 = 0
    
    init(direct: Int64 = 0, peer: Int64 = 0) {
        self.direct = direct
        self.peer = peer
    }
}
